### PR TITLE
Setup database with Post table and methods 

### PR DESCRIPTION
**Pull request by kevinzhuang10**
Create DB method for Post model

Create test api route for inserting and reading  from DB

Create Post table using Knex migration

Setup Front-end Scaffold

### CODE REVIEW

File: config/custom-environment-variables.json [hunk 1]
```diff
@@ -4,18 +4,20 @@
       "clientID": "GOOGLE_CLIENT_ID",
       "clientSecret": "GOOGLE_CLIENT_SECRET",
       "callbackURL": "GOOGLE_CALLBACK_URL"
-    },
+    }
+    // ,
```
Comment (lazdinst) [inline config/custom-environment-variables.json:8]: Not sure if these comment slashes will cause issues.

**Review by lazdinst — APPROVED** [review]
Approved. Talis Lazdins

**Review by Sam0064 — APPROVED** [review]
Approved.

File: db/DB_README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+## Schema Creation / Modification
```
Comment (olegtrofimov1) [inline db/DB_README.md:1]: 👍

File: server/routes/api.js [hunk 3]
```diff
@@ -11,4 +12,26 @@ router.route('/')
     res.status(201).send({ data: 'Posted!' });
   });
 
+router.route('/test')
```
Comment (olegtrofimov1) [inline server/routes/api.js:15]: I get that you are just testing your DB code here, but you should not include anything in your code submissions that is development code not adding to the product.

**Review by olegtrofimov1 — COMMENTED** [review]
Where are the tests?

**Pull request merged**